src/components/Footer/Footer.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import SocialFollow from "./SocialFollow";
+import "./style.css";
```

Comment:
No need of importing style.css again as it is already imported in the App component